docs/data-sources/cdn.md

```diff
@@ -137,7 +136,7 @@ Read-Only:
 Read-Only:
 
 - `ssl_id` (String) ID (UUID) of the SSL certificate
-- `type` (String) Available values: instantSsl, none, SNI
+- `type` (String) Possible values: instantSsl, none, SNI, SAN
```

Comment:
ten SAN imho uz vubec nenabizime

Reply:
Why we have it in the api then? https://client.cdn77.com/support/api-reference/v3/cdn-resources#edit-cdn-resource